tests/run.sh

```diff
@@ -174,6 +174,9 @@ if [ -n "$DO_COVERAGE" ] && [ "$DO_COVERAGE" != 0 ]; then
     sed "1 s~^~require('luacov.runner')('$source_dir/.luacov'); \0~" \
         "$RC_FILE" > "$tmp_files/awesomerc.lua"
     RC_FILE=$tmp_files/awesomerc.lua
+
+    # Use xresources theme for more coverage.
+    sed -i 's~default/theme~xresources/theme~' "$RC_FILE"
```

Comment:
> will it be added to the coverage?

Yup, coverage tracing uses debug hooks and those also run from `dofile()`'d things. I'm just not sure if there could be problems because of relative vs absolute paths.

> and does it makes any sense at all?

I'm not sure. Perhaps not...?